processor/src/coins/monero.rs

```diff
@@ -536,7 +549,7 @@ impl Coin for Monero {
 
   #[cfg(test)]
   async fn get_fee(&self) -> Self::Fee {
-    self.rpc.get_fee().await.unwrap()
+    self.rpc.get_fee(0).await.unwrap()
```

Comment:
Is 0 highest or lowest priority? Lowest, right?

Reply:
0 is lowest. FWIW, upon working through the v17 fixes, I noticed the CLI, RPC, and GUI all end up defaulting this to 1 in the normal case thanks to a separate code path in those wallets that adjusts the priority down to 1 if there is no backlog in the pool. I defaulted to 1 in the latest and included a TODO to implement an `adjust_priority`